circuit/src/commonMain/kotlin/com/slack/circuit/Navigator.kt

```diff
@@ -11,9 +11,13 @@ public interface Navigator {
 
   public fun pop(): Screen?
 
+  /** Clear the existing backstack of [Screens][Screen] and navigate to [newRoot]. */
```

Comment:
Let's flush out this doc a little more, perhaps with an example of when you'd want to use this

Reply:
[7364a74](https://github.com/slackhq/circuit/pull/339/commits/7364a7455320a8928a081831d454146ec01d3ab3)